### ISSUE TITLE
When a new course is added with a start date in the future, nothing is run and a day is missed

### ISSUE THREAD

**Issue by jonespm** [tl-its-umich-edu/my-learning-analytics#1138]
Thank you for contributing to this project!

- Make sure to search the [issues](https://github.com/tl-its-umich-edu/my-learning-analytics/issues) for duplicates first!

### Expected behavior (A description of what you expected to happen) : 
When a course (or courses) are added with a future start date (after current date) it should really not consider needing to populate this as a new course and skip it. 

### Describe the bug (Tell us what happens instead of the expected behavior) :
If it doesn't it, when this future course is added, the system will do an resources accessed update taking this date as the the update date. This results in nothing getting updated for that date but the record of courses being updated still being made.

### Steps to Reproduce :
1. Add a course with a future start date (after today)
2. Run the cron
3. It should skip this course, but it's using this date in the future as the update date.

**Comment by jonespm** [comment]
An example of a course in the future is 415293. You want to add this and a course in the past like 428580.

On the first run, it should run from the course in the past.
On the next run is should run from the date these were both last updated.

**Comment by ssciolla** [comment]
> Note: if you run the cron and it fails (like the credentials are bad) you either have to remove/readd the course or clear out the data last updated in the database, this is another bug

@jonespm, is this documented?

**Comment by jonespm** [comment]
Not exactly, I was hoping to fix this specific problem by the end of the week though in #1130

**Comment by pushyamig** [comment]
1. I created a course start date in future 17700000000434903 and also prior to the cron run I created some events for the course in past (17700000000429051) and I saw the for the 17700000000429051 new events are brought in 
2. There was already a course with future date in the test instance and things worked as expected.
3. I add a course from fall term 17700000000314402 and I see cron pull past events.
4. date_last_updated field is acting correctly 

I felt ` Deleting all records in resource_access after 2021-01-07 13:43:52.105049+00:00` this message in the log was bit confusing with flow but Functionally this seems to work

From what I have tested Test passes.
